Framework/DataHandling/inc/MantidDataHandling/LoadILLDiffraction.h

```diff
@@ -92,8 +92,8 @@ class MANTID_DATAHANDLING_DLL LoadILLDiffraction
   void setSampleLogs();
   void computeThetaOffset();
 
-  double
-      m_offsetTheta; ///< the 2theta offset for D20 to account for dead pixels
+  double m_offsetTheta{
+      0.};           ///< the 2theta offset for D20 to account for dead pixels
```

Comment:
The doxygen comment could be on the preceding line to get this formatted correctly.